plugins/modules/aci_tenant_span_src_group_src.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, Akini Ross (@akinross)
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_tenant_span_src_group_src
+short_description: Manage SPAN source group sources (span:Src)
+description:
+- Manage SPAN source group sources on Cisco ACI fabrics.
+options:
+  name:
+    description:
+    - The name of the Span source.
+    type: str
+  description:
+    description:
+    - The description for Span source.
+    type: str
+    aliases: [ descr ]
+  src_group:
+    description:
+    - The name of the Span source group.
+    type: str
+    aliases: [ name ]
```

Comment:
This will conflict with the first name option

Reply:
ok will correct

Reply:
Not fixed